DroneController/app/src/main/java/com/dotscene/dronecontroller/SystemStateFragment.java

```diff
@@ -223,6 +238,22 @@ public void onClick(@NonNull View widget) {
     s.setSpan(clickableSpan, 0, s.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
     rosErrorText.setText(s);
 
+    // Setup warning notifications
+    warning_notification_sdf = new SimpleDateFormat("HH:mm:ss");
+    // Create the NotificationChannel, but only on API 26+ because
+    // the NotificationChannel class is new and not in the support library
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+      CharSequence name = "Warnings";
+      String description = "Warnings that pop up while using the dotcube.";
+      int importance = NotificationManager.IMPORTANCE_DEFAULT;
```

Comment:
```suggestion
      int importance = NotificationManager.IMPORTANCE_HIGH;
```
It feels like we should be very obnoxious about these warnings.

Reply:
Done